### DISCUSSION TITLE
Allow new signups Option is not effective

### DISCUSSION THREAD

**Discussion by hellrazorx64** [dani-garcia/vaultwarden#2615] [Q&A]
Tried with or without,
Test user is still able to signup, get email confirmation, then log into their personal vault.

![image](https://user-images.githubusercontent.com/82125317/178543886-42014f6a-87d4-4a8b-afa0-10809297b67b.png)

Version 2.28.1

**Comment by BlackDex** [comment]
Please provide the support string as requested which you can generate via the admin interface on the Diagnostics page.

**Reply by hellrazorx64** [reply]
I just realized that the option is overrid by Domain Whitelisting.

I expected the following behavior:
Allow new signups (disable) = Overrall denial of signup regardless of white listing

At first I saw whitelisting as a supplementary validation method to NOT let everybody invite anyone from anywhere to the server since user invitation outside whitelisting isn't possible.

Probably very debatable. But I think further options could be accessible.

For my two cents, i'd like to
- Let any white listed domain members to signup
- Let Organization admin invite anyone even from outside domain.

I'll let the White list on for now since we own a domain and we are still on eploying phase, thanks for making me dig.

**Reply by BlackDex** [reply]
If you want to invite someone from outside that whitelist, i suggest to invite them via the /admin interface, since that overrules all restrictions.

**Comment by stephanfriedrich** [comment]
Hi thx for your awesome vaultwarden rebuild. 

currently i also disabled the "Allow new signups" with Docker-Env-Variables and also tried on admin-interface, but unfortunately it wont work.
the button "create account" is still visible on "/login" page.

i also have no settings in the whitelist as described here.

has this option to disable user registrations been tested ?

have a nice day.
greetings

**Reply by Masgalor** [reply]
See https://github.com/dani-garcia/vaultwarden/discussions/3393
